### PR TITLE
doc(helm): add helm values to website

### PR DESCRIPTION
**Pull request by ChanYiLin**
ref: https://github.com/longhorn/longhorn/issues/5455

### CODE REVIEW

File: content/docs/1.6.0/references/helm-values.md
Comment (shuo-wu) [inline content/docs/1.6.0/references/helm-values.md:164]: NIT: make this a separate table and mention when users need to care it

File: content/docs/1.6.0/references/helm-values.md
Comment (shuo-wu) [inline content/docs/1.6.0/references/helm-values.md:144]: Can we have a better organization for the distro-related settings? e.g.,
```
### OS/Kubernetes Distro Settings
#### Opensift Settings
#### Talos Settings
```

File: content/docs/1.6.0/references/helm-values.md
Comment (innobead) [reply content/docs/1.6.0/references/helm-values.md:111]: Good suggestion.

For any `non-primitive` values, it's helpful to have examples.

File: content/docs/1.6.0/references/helm-values.md
Comment (innobead) [inline content/docs/1.6.0/references/helm-values.md:88]: The format of the enum values can be consistent. I see some of them are seperated by comma.

File: content/docs/1.6.0/references/helm-values.md
Comment (innobead) [inline content/docs/1.6.0/references/helm-values.md:136]: Missed the description.

File: content/docs/1.6.0/references/helm-values.md
Comment (innobead) [inline content/docs/1.6.0/references/helm-values.md:159]: This is tricky. We probably need to remove this setting, because currently, we don't suggest installing longhorn outside longhorn-system ns.

**Review by innobead — CHANGES_REQUESTED** [review]
Some feedback. In general, it looks good. However, we need to ensure the descriptoin stype consistent in each value setting.

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:95]: The **Default** here are taken from `values.yaml`
However, we left black in the `values.yaml`, so the generator showed nil for the value
```
csi:
  # -- Specify kubelet root-dir. Leave blank to autodetect
  kubeletRootDir: ~
  # -- Specify replica count of CSI Attacher. Leave blank to use default count: 3
  attacherReplicaCount: ~
  # -- Specify replica count of CSI Provisioner. Leave blank to use default count: 3
  provisionerReplicaCount: ~
  # -- Specify replica count of CSI Resizer. Leave blank to use default count: 3
  resizerReplicaCount: ~
  # -- Specify replica count of CSI Snapshotter. Leave blank to use default count: 3
  snapshotterReplicaCount: ~
```

Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:95]: After discussion, remove **Default** for this csi part from the readme

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:111]: We actually have example in `values.yaml` comment
https://github.com/longhorn/longhorn/blob/master/chart/values.yaml#L192
So when the users want to change the setting, they can follow the comment. Do you think it is enough?

```
longhornManager:
  log:
    ## Allowed values are `plain` or `json`.
    format: plain
  priorityClass: ~
  tolerations: []
  ## If you want to set tolerations for Longhorn Manager DaemonSet, delete the `[]` in the line above
  ## and uncomment this example block
  # - key: "key"
  #   operator: "Equal"
  #   value: "value"
  #   effect: "NoSchedule"
  nodeSelector: {}
  ## If you want to set node selector for Longhorn Manager DaemonSet, delete the `{}` in the line above
  ## and uncomment this example block
  #  label-key1: "label-value1"
  #  label-key2: "label-value2"
  serviceAnnotations: {}
  ## If you want to set annotations for the Longhorn Manager service, delete the `{}` in the line above
  ## and uncomment this example block
  #  annotation-key1: "annotation-value1"
  #  annotation-key2: "annotation-value2"
```

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:164]: Fixed, and also mention that these settings are needed when installing Longhorn in air-gap environment
with pointing to the doc 
https://longhorn.io/docs/1.5.1/advanced-resources/deploy/airgap/

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:14]: Fixed

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:16]: fixed.

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:19]: Fixed.

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:86]: Yes, you are right
https://github.com/longhorn/longhorn/pull/6393
In this ticket there is also same discussion
I will fix it in this PR

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:88]: Fixed

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md:136]: Fixed.

File: content/docs/1.6.0/references/helm-values.md
Comment (innobead) [reply content/docs/1.6.0/references/helm-values.md]: It's better to have that in doc, but I think it's fine for now. Let's make it simple first.

File: content/docs/1.6.0/references/helm-values.md
Comment (innobead) [inline content/docs/1.6.0/references/helm-values.md:137]: Capitalize each description and make it consistent with others like `priorityClass` -> `Priority class`

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, but just one comment required to fix.

File: content/docs/1.6.0/references/helm-values.md
Comment (ChanYiLin) [reply content/docs/1.6.0/references/helm-values.md]: fixed.

**Review by innobead — APPROVED** [review]

**Review by jillian-maroket — COMMENTED** [review]
I reviewed the descriptions and section introductions. I implemented a new wording convention, which will be documented on a Confluence page when I have more bandwidth. Let me know if the changes work.

**Comment by innobead** [comment]
@jillian-maroket, please create a PR to update the doc with your previous suggestions. After the changes are merged, @ChanYiLin will then update `https://github.com/longhorn/longhorn/blob/master/chart/{README.md, values.yaml questions.yaml}`, because they are the source of truth of this doc.

In the future, @jillian-maroket should review the files in longhorn/longhorn instead, because the doc here will be generated from their changes. 

ref: https://github.com/longhorn/longhorn/wiki/Longhorn-Setting-Update

**Pull request merged**
